### PR TITLE
33 There is no guarantee that users can buy hardCap amount of tokens

### PR DESCRIPTION
**Pull request by decadanse**
closes <https://github.com/PrimeDAO/launch-contracts/issues/33>

"seedRemainder -= seedAmount; 
       feeRemainder -= feeAmount;

As the initial value of  seedRemainder is calculated on the basis of the initial price rather than the actual price at which tokens are being sold, this limit can be reached at any time during the sale - also before the hard cap, or even soft cap, is reached."

connected to https://github.com/PrimeDAO/launch-contracts/pull/28
as "fix bug with feeRemainder" have fixes so feeRemainder can not go < 0

### CODE REVIEW

**Review by arseneeth — CHANGES_REQUESTED** [review]
I don't quite understand why this PR is only a comment update

File: contracts/seed/Seed.sol [hunk 1]
```diff
@@ -389,6 +389,8 @@ contract Seed {
 
         if (!isFunded) {
             require(
+                // seedAmountRequired is an amount which is needed to be selled
+                // So when it's reached, for others will their balance be bigger or not - doesn't matter anymore.
```
Comment (BritikovKI) [inline contracts/seed/Seed.sol:393]: Nice comment

**Review by BritikovKI — APPROVED** [review]

**Comment by BritikovKI** [comment]
> I don't quite understand why this PR is only a comment update

Because it's a feature, not a bug, comment is just a clarification here for the auditors

File: contracts/seed/Seed.sol (hunk 1 above)
Comment (arseneeth) [reply contracts/seed/Seed.sol:393]: *sold not selled

**Review by arseneeth — CHANGES_REQUESTED** [review]

**Pull request merged**